masonry/src/widget/widget.rs

```diff
@@ -220,9 +220,11 @@ pub trait Widget: AsAny {
         None
     }
 
-    // TODO - Document
-    // TODO - Add &UpdateCtx argument
-    fn get_cursor(&self) -> CursorIcon {
+    /// Return the cursor icon for this widget.
```

Comment:
We should document when this will be called here as well, i.e. something like `when the mouse cursor moves. It may also be called due to cursor_icon_changed, for cases when the state under the cursor might have changed (consider for example a link moving under the cursor due to a keypress in a rich text box)`